src/dymoprint/lib/dymo_labeler.py

```diff
@@ -219,40 +213,41 @@ def _get_status(self):
         self._status_request()
         return self._send_command()
 
-    def print_label(self, lines: list[list[int]], margin_px=DEFAULT_MARGIN_PX):
+    def print_label(self, lines: list[list[int]]):
         """Print the label described by lines.
 
         Automatically split the label if it's larger than maxLines.
         """
         while len(lines) > self._maxLines + 1:
-            self._raw_print_label(lines[0 : self._maxLines], margin_px=0)
+            self._raw_print_label(lines[0 : self._maxLines])
             del lines[0 : self._maxLines]
-        self._raw_print_label(lines, margin_px=margin_px)
+        self._raw_print_label(lines)
 
-    def _raw_print_label(self, lines: list[list[int]], margin_px=DEFAULT_MARGIN_PX):
+    def _raw_print_label(self, lines: list[list[int]]):
         """Print the label described by lines (HLF)."""
         # Here used to be a matrix optimization code that caused problems in issue #87
         self._tape_color(0)
         for line in lines:
             self._line(line)
-        if margin_px > 0:
-            self._skip_lines(margin_px * 2)
         self._status_request()
         status = self._get_status()
         LOG.debug(f"Post-send response: {status}")
 
 
 class DymoLabeler:
     device: DetectedDevice
-    margin_px: int
     tape_size_mm: int
 
+    LABELER_DISTANCE_BETWEEN_PRINT_HEAD_AND_CUTTER_MM = 8.1
+    LABELER_PRINT_HEAD_HEIGHT_MM = 8.2
+    SUPPORTED_TAPE_SIZES_MM = (19, 12, 9, 6)
+    DEFAULT_TAPE_SIZE_MM = 12
+
     def __init__(
         self,
-        margin_px: int = DEFAULT_MARGIN_PX,
         tape_size_mm: int = DEFAULT_TAPE_SIZE_MM,
     ):
-        self.margin_px = margin_px
+        assert tape_size_mm in self.SUPPORTED_TAPE_SIZES_MM
```

Comment:
Unless there's something else forcing `tape_size_mm` to be a supported tape size, this should be a `ValueError` and not an `AssertionError`. Note that Python strips assert statements when run in optimized mode, so assert statements shouldn't be used for any sort of error handling.

(Note that given our installation instructions probably 100% of our users will not be running in optimized mode, so my statement is more on principle than in practice; semantically `ValueError` is more accurate than `AssertionError`.)